io.github.vinser.pixyne.yml

```diff
@@ -0,0 +1,34 @@
+app-id: io.github.vinser.pixyne
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.golang
+command: pixyne
+
+finish-args:
+    - --share=ipc 
+    - --socket=x11
+    - --device=dri
+    # We want user fs access so we can read and write folders with jpeg photo files, save user preferences
+    - --filesystem=home
```

Comment:
Where is preferences being saved? They should be relative to `XDG_CONFIG_HOME` not `$HOME`.

Do you support filechooser portals? https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.FileChooser.html

Reply:
Pixyne app is based on cross-platform [Fyne UI toolkit](https://github.com/fyne-io/fyne) written in Go. Portals are not yet implemented in Fyne, so I can't use them.  On Linux OSes they keep `preferences.json` in `/home/\<user\>/.config/fyne/io.github.vinser/pixyne` directory. 
In addition, the user must be able to read and save jpeg photo files in folders in his home directory
That's why I wrote `--filesystem=home`

Reply:
> On Linux OSes they keep preferences.json in /home/<user>/.config/fyne/io.github.vinser/pixyne directory.

According to https://github.com/fyne-io/fyne/commit/7726efeb9703e3ad0ccdbf7149630fce743a25ac it shouldn't be the case (it's actually good)

Reply:
sa for the photos, `xdg-pictures` should be good enough.